src/main/java/com/theelixrlabs/healthcare/service/PatientService.java

```diff
@@ -25,44 +27,44 @@ public PatientService(PatientRepository patientRepository) {
     /**
      * Validates the PatientDTO before adding details.
      *
-     * @param patientDTO    The data transfer object containing patient information.
-     * @throws CustomException    If validation fails (e.g., empty first name or invalid characters).
+     * @param patientDTO The data transfer object containing patient information.
+     * @throws CustomException If validation fails (e.g., empty first name or invalid characters).
      */
     private void validatePatientDTO(PatientDTO patientDTO) throws CustomException {
 
         //Validate first name
-        if (patientDTO.getFirstName().isEmpty()) {
+        if (patientDTO.getPatientFirstName().isEmpty()) {
             throw new CustomException(PatientConstants.FIRST_NAME_SHOULD_NOT_BE_EMPTY);
-        } else if (!patientDTO.getFirstName().matches(PatientConstants.ALPHA_CHARACTERS_ONLY_REGEX)) {
+        } else if (!patientDTO.getPatientFirstName().matches(PatientConstants.ALPHA_CHARACTERS_ONLY_REGEX)) {
             throw new CustomException(PatientConstants.INVALID_FIRST_NAME);
         }
 
         //Validate last name
-        if (patientDTO.getLastName().isEmpty()) {
+        if (patientDTO.getPatientLastName().isEmpty()) {
             throw new CustomException(PatientConstants.LAST_NAME_SHOULD_NOT_BE_EMPTY);
-        } else if (!patientDTO.getLastName().matches(PatientConstants.ALPHA_CHARACTERS_ONLY_REGEX)) {
+        } else if (!patientDTO.getPatientLastName().matches(PatientConstants.ALPHA_CHARACTERS_ONLY_REGEX)) {
             throw new CustomException(PatientConstants.INVALID_LAST_NAME);
         }
     }
 
     /**
      * Creates a new patient based on the provided PatientDTO.
      *
-     * @param patientDTO    The data transfer object containing patient information.
+     * @param patientDTO The data transfer object containing patient information.
      * @return The PatientDTO of the newly created patient, with ID populated.
-     * @throws CustomException    If validation fails or if the Aadhaar number already exists.
+     * @throws CustomException If validation fails or if the Aadhaar number already exists.
      */
     public PatientDTO addPatientDetails(PatientDTO patientDTO) throws CustomException {
 
         //Validate the incoming patientDTO
         validatePatientDTO(patientDTO);
 
         //Format Aadhaar number
-        String aadhaarNumber = patientDTO.getAadhaarNumber();
+        String aadhaarNumber = patientDTO.getPatientAadhaarNumber();
         String formattedAadhaarNumber = String.format(PatientConstants.AADHAAR_FORMAT_PATTERN, aadhaarNumber.substring(0, 4), aadhaarNumber.substring(4, 8), aadhaarNumber.substring(8, 12));
 
         //Check if aadhaar number already exists.
-        if (patientRepository.findByAadhaarNumber(formattedAadhaarNumber).isPresent()) {
+        if (patientRepository.findByPatientAadhaarNumber(formattedAadhaarNumber).isPresent()) {
```

Comment:
Make the return value of findByPatientAadhaarNumber() optional to accomodate the possibility of it returning null.